src/social/components/Comment/index.js

```diff
@@ -206,7 +206,7 @@ const Comment = ({ readonly = false, commentId, currentUserId, userRoles }) => {
       handleEdit={handleEdit}
       handleDelete={deleteComment}
       isEditing={isEditing}
-      queryMentionees={queryMentionees.filter((q) => !q.isGlobalBan)}
+      queryMentionees={queryMentionees}
```

Comment:
Isn't that something that should not be extracted as a function too? That might result in some people showing as links even if they are banned.